source/webapp/sites/all/modules/custom/checkbook_custom_template/templates/dept_revenue.tpl.php

```diff
@@ -20,10 +20,10 @@
 ?>
 <?php require_once(realpath(drupal_get_path('module', 'data_controller')) .'/common/object/converter/handler/PHP2Json.php');
 if (module_exists('widget_highcharts')){
-    drupal_add_js(drupal_get_path('module', 'widget_highcharts') .'/highcharts/7.1.1/code/highcharts.js', array('group'=>JS_LIBRARY,'weight'=>-1));
+    drupal_add_js(drupal_get_path('module', 'widget_highcharts') .'/highcharts/8.2.2code/highcharts.js', array('group'=>JS_LIBRARY,'weight'=>-1));
```

Comment:
I think there is a '/' missing here

Reply:
I meged code from 'NYCHA' branch. Let me fix in it all branches.